core/trino-main/src/test/java/io/trino/execution/TestNodeScheduler.java

```diff
@@ -135,9 +135,9 @@ public void setUp()
     private void setUpNodes()
     {
         nodeManager.addNodes(
-                new InternalNode("other1", URI.create("http://10.0.0.1:11"), NodeVersion.UNKNOWN, false),
-                new InternalNode("other2", URI.create("http://10.0.0.1:12"), NodeVersion.UNKNOWN, false),
-                new InternalNode("other3", URI.create("http://10.0.0.1:13"), NodeVersion.UNKNOWN, false));
+                new InternalNode("other1", "other1", URI.create("http://10.0.0.1:11"), NodeVersion.UNKNOWN, false),
```

Comment:
nit: maybe avoid using same value for nodeId and instanceId in tests as this does not mimic what you would get in production.

Reply:
that was pretty boring manual change. there were too many places to update for me to be creative.